src/clients/HubPoolClient.ts

```diff
@@ -70,17 +70,19 @@ type L1TokensToDestinationTokens = {
 };
 
 // Temporary type for v2 -> v3 transition. @todo: Remove.
-export type V2PartialDepositWithBlock = Pick<
+type V2PartialDepositWithBlock = Pick<
   V2DepositWithBlock,
-  "originChainId" | "destinationChainId" | "originToken" | "amount" | "quoteTimestamp" | "blockNumber"
+  "originChainId" | "originToken" | "amount" | "quoteTimestamp"
 >;
 
 // Temporary type for v2 -> v3 transition. @todo: Remove.
-export type V3PartialDepositWithBlock = Pick<
+type V3PartialDepositWithBlock = Pick<
   V3DepositWithBlock,
-  "originChainId" | "destinationChainId" | "inputToken" | "inputAmount" | "quoteTimestamp" | "blockNumber"
+  "originChainId" | "inputToken" | "inputAmount" | "quoteTimestamp"
 >;
 
+export type LpFeeRequest = (V2PartialDepositWithBlock | V3PartialDepositWithBlock) & { paymentChainId: number };
```

Comment:
`paymentChainId` selected because this also covers slow fills, in which case there's no `repayment` per se.